docs/home/api-info/webhooks.mdx

```diff
@@ -104,6 +105,44 @@ as the data becomes available after initiating a document query.
 
 You can see Webhook details specific to the Medical API on [this page](/medical-api/more-info/webhooks).
 
+### Meta data
+
+Webhook updates will contain meta information.
+
+Example payload:
+
+```json
+{
+  "meta": {
+    "messageId": "<message-id>",
+    "when": "<date-time-in-utc>",
+    "type": "devices.health-data"
+  }
+}
+```
+
+The format follows:
+
+<ResponseField name="meta" required>
+  Metadata about the message.
+  <Expandable title="meta type property details">
+
+    <ResponseField name="messageId" type="string" required>
+      The ID for this message from Metriport, useful for debugging purposes only;
+    </ResponseField>
+
+    <ResponseField name="when" type="string" required>
+      The timestamp when this message was originally sent, formatted as [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601)
+      (example: `2022-12-24T00:46:05.413Z`)
+    </ResponseField>
+
+    <ResponseField name="type" type="DAPIWebhookType | MAPIWebhookType" required>
+      The type of the webhook message. Can be one of: `devices.<subtype>`, `medical.<subtype>`.
```

Comment:
let's be explicit, this is obvious to us but not to new people using the product:

"The type of the webhook message. If coming from the Devices API, this will be formatted `devices.<subtype>`, and if coming from the Medical API it will be formatted `medical.<subtype>`."